applications/daily_tests/automatic_sync_test.js

```diff
@@ -9,38 +9,72 @@ const SyncType = {
 };
 
 async function main() {
-  let syncType;
+  let syncType, forced;
   if (process.argv.includes("pruned")) {
     syncType = SyncType.Pruned;
   } else {
     syncType = SyncType.Archival;
   }
 
-  const { blockHeight, timeDiffMinutes, blockRate } = await run({
-    log: "./logs/base-node.log",
-    syncType,
-  });
+  // Check for forced index.
+  let index = process.argv.findIndex((element) => element === "forced");
```

Comment:
I think now that we are using cli args with arguments, use yargs like  automated_recovery_test

Reply:
Done.